python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -60,6 +60,9 @@ def main(config_args):
     config = GSConfig(config_args)
     config.verify_arguments(True)
 
+    # TODO(IN): Support node classification/regression for wholegraph
+    if use_wholegraph(config.part_config):
```

Comment:
Why it is not supported?
Right now we store labels in dgl graph, so np should be fine.

Reply:
right! Reverting.